cmd/flux/install_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestInstallNoArgs(t *testing.T) {
+	cmd := cmdTestCase{
+		args:            "install",
+		wantError:       false,
+		testClusterMode: ExistingClusterMode,
```

Comment:
Each test should run in isolation on its own control plane, I don’t think ExistingClusterMode should be specified in code at all. I see the ExistingClusterMode as an env var that a contributor would set to debug tests locally. As it is now, if someone runs the tests, and the kubeconfig is set to a production cluster, we’ll wipe it clean causing a major incident…

Reply:
The main reason is that a testEnv has only the control plane, which is not enough for us to use.
Commands like "flux install" requires a fully functional cluster.

Instead of this we could spinning up a KinD cluster with a random unique context name and make sure our tests will run against that only.

 wdyt?

Reply:
Your makefile changes look great now! Thanks